test/api/v3/integration/groups/GET-groups.test.js

```diff
@@ -0,0 +1,35 @@
+import {
```

Comment:
Shouldn't this test look more like: https://github.com/HabitRPG/habitrpg/blob/8a546c7ec3d41acd606991aca1f78c813f0ff64b/test/api/v2/groups/GET-groups.test.js

Adjusted of course for the way v3 differs from v2.